nrf_security/CMakeLists.txt

```diff
@@ -18,59 +18,51 @@ endif()
 
 include(${CMAKE_CURRENT_LIST_DIR}/cmake/nrf_security_debug.cmake)
 
-if(CONFIG_BUILD_WITH_TFM)
-  nrf_security_debug("TF-M is enabled, nRF Security will build as part of TF-M")
+#
+# Populate ARM_MBEDTLS_PATH
+#
+get_mbedtls_dir(ARM_MBEDTLS_PATH)
+
+# Neither pure Zephyr nor NS-build can set key ID encodes owner
+set(MBEDTLS_PSA_CRYPTO_KEY_ID_ENCODES_OWNER       False)
```

Comment:
How can this be set to false in a TF-M build?

Reply:
I don't know what this is. I just refactored it.

It was set to False already.

Reply:
Please Update comment

```
# MBEDTLS_PSA_CRYPTO_KEY_ID_ENCODES owner can't be set for Zephyr builds
# or when MBEDTLS_USE_PSA_CRYPTO is set (e.g. for TLS/DTLS and x.509 support)
# Note: This configuration is internal and may be removed with a new mbed TLS version
```